projects/core/src/lib/operators/dialogs/expression-operator/expression-operator.component.ts

```diff
@@ -220,25 +205,15 @@ export class ExpressionOperatorComponent implements AfterViewInit {
                                 mapNoData,
                             },
                             sources: {
-                                a: operators[0],
-                                b: operators.length >= 2 ? operators[1] : undefined,
-                                c: operators.length >= 3 ? operators[2] : undefined,
-                                d: operators.length >= 4 ? operators[3] : undefined,
-                                e: operators.length >= 5 ? operators[4] : undefined,
-                                f: operators.length >= 6 ? operators[5] : undefined,
-                                g: operators.length >= 7 ? operators[6] : undefined,
-                                h: operators.length >= 8 ? operators[7] : undefined,
+                                raster: inputWorkflow.operator,
                             },
                         } as ExpressionDict,
-                    };
+                    } as WorkflowDict;
 
                     return this.projectService.registerWorkflow(workflow);
                 }),
                 mergeMap((workflowId: UUID) => {
-                    const symbology$: Observable<RasterSymbology> = this.symbologyCreator.symbologyForRasterLayer(
-                        workflowId,
-                        rasterLayers[0],
-                    );
+                    const symbology$: Observable<RasterSymbology> = this.symbologyCreator.symbologyForRasterLayer(workflowId, rasterLayer);
```

Comment:
don't know if this is the right line but there is an issue when selecting "symbology like input": If the input has more bands then the output (and another then the first band is currently displayed for the input) the symbology of the output references a non existing band and throws errors.

Reply:
good catch 4664e93b4ddcba723a2635b765083a740861ed1a